AdminServer/appscale/admin/utils.py

```diff
@@ -497,3 +502,105 @@ def queues_from_dict(payload):
     queues[name] = queue
 
   return {'queue': queues}
+
+
+class _PersistentWatch(object):
+
+  def __init__(self):
+
+    class CompliantLock(object):
+      """
+      A container for three pieces which makes possible to
```

Comment:
Wording is a bit confusing. Could we change it to "A container which allows to organize compliant locking of zk_node by making sure:"